pkg/azuredx/azureauth/auth.go

```diff
@@ -0,0 +1,24 @@
+// Package azureauth implements Azure authorization.
+package azureauth
+
+import (
+	"errors"
+	"strings"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+)
+
+// BearerToken extracts the client's token.
+func BearerToken(req *backend.QueryDataRequest) (accessToken string, err error) {
+	s, ok := req.Headers["Authorization"]
+	if !ok {
+		return "", errors.New("no HTTP Authorization value")
+	}
+
+	const prefix = "Bearer "
```

Comment:
When using `X-ID-Token`, then we can just return it.
```suggestion
	return s
```